lib/_http_incoming.js

```diff
@@ -47,7 +47,7 @@ function IncomingMessage(socket) {
   // response (client) only
   this.statusCode = null;
   this.statusMessage = null;
-  this._client = socket; // deprecated
+  this.client = socket;
```

Comment:
@mscdex 
Your comment looks like you thought that I suggested deprecating one of those.
I just suggested to change the deprecation message for `.client` to mention just one of those.